src/app/home/pages/PlansPage/components/Wizard/WizardFormik.tsx

```diff
@@ -20,129 +23,136 @@ const WizardFormik: React.FunctionComponent<IWizardFormikProps> = ({
+}: IWizardFormikProps) => {
+  const dispatch = useDispatch();
+  return (
+    <Formik<IFormValues>
+      initialValues={initialValues}
+      validate={(values: IFormValues) => {
+        const errors: { [key in keyof IFormValues]?: string } = {}; // TODO figure out why using FormikErrors<IFormValues> here causes type errors below
 
-      if (!values.planName) {
-        errors.planName = 'Required';
-      } else if (!utils.testDNS1123(values.planName)) {
-        errors.planName = utils.DNS1123Error(values.planName);
-      } else if (values.planName.length < 3 || values.planName.length > 63) {
-        errors.planName = 'The plan name should be between 3 and 63 characters long.';
-      } else if (
-        !isEdit &&
-        !currentPlan &&
-        planList.some((plan) => plan.MigPlan.metadata.name === values.planName)
-      ) {
-        errors.planName =
-          'A plan with that name already exists. Enter a unique name for the migration plan.';
-      }
-
-      if (!values.sourceCluster) {
-        errors.sourceCluster = 'Required';
-      }
-      if (!values.selectedNamespaces || values.selectedNamespaces.length === 0) {
-        errors.selectedNamespaces = 'Required';
-      }
-      if (!values.selectedPVs || values.selectedPVs.length === 0) {
-        errors.selectedPVs = 'Required';
-      }
+        if (!values.planName) {
+          errors.planName = 'Required';
+        } else if (!utils.testDNS1123(values.planName)) {
+          errors.planName = utils.DNS1123Error(values.planName);
+        } else if (values.planName.length < 3 || values.planName.length > 63) {
+          errors.planName = 'The plan name should be between 3 and 63 characters long.';
+        } else if (
+          !isEdit &&
+          !currentPlan &&
+          planList.some((plan) => plan.MigPlan.metadata.name === values.planName)
+        ) {
+          errors.planName =
+            'A plan with that name already exists. Enter a unique name for the migration plan.';
+        }
 
-      if (!values.targetCluster) {
-        errors.targetCluster = 'Required';
-      }
-      if (!values.selectedStorage) {
-        errors.selectedStorage = 'Required';
-      }
-      const hasDuplicateMapping = values.editedNamespaces.find((ns, index) => {
-        const editedNSName = values?.currentTargetNamespaceName?.name;
-        const editedNSNameID = values?.currentTargetNamespaceName?.id;
-        return (
-          (editedNSName === ns.newName && editedNSNameID !== ns.id) ||
-          (editedNSName === ns.oldName && editedNSNameID !== ns.id)
-        );
-      });
+        if (!values.sourceCluster) {
+          errors.sourceCluster = 'Required';
+        }
+        if (!values.selectedNamespaces || values.selectedNamespaces.length === 0) {
+          errors.selectedNamespaces = 'Required';
+        }
+        if (
+          values.migrationType.value !== 'scc' &&
```

Comment:
This is the only validation change made - the rest are formatting changes from prettier :( -- Checks if the plan type is SCC & if so, allow empty selectedPVs.